copy/copy.go

```diff
@@ -646,6 +688,27 @@ func (c *copier) copyOneImage(ctx context.Context, policyContext *signature.Poli
 	// If encrypted and decryption keys provided, we should try to decrypt
 	ic.diffIDsAreNeeded = ic.diffIDsAreNeeded || (isEncrypted(src) && ic.c.ociDecryptConfig != nil) || ic.c.ociEncryptConfig != nil
 
+	options.OptimizeDestinationImageAlreadyExists = true // TODO: PR: Remove forced enablement for PR testing
+	// If enabled, fetch and compare the destination's manifest. And as an optimization skip updating the destination iff equal
```

Comment:
This does not make much sense just above the `isSigned` computation.

It might make sense to document the purpose of the whole section of the code, OTOH maybe just the `if options.Optimize…` (if it becomes top-level, as suggested below) with no comment is enough for that.